code/modules/customitems/item_defines.dm

```diff
@@ -2268,7 +2268,82 @@ All custom items with worn sprites must follow the contained sprite system: http
 		return
 
 	SEND_SIGNAL(D, COMSIG_ITEM_UPDATE_STATE, D)
+  
+/obj/item/fluff/nasira_burner
+	name = "adhomian incense burner"
+	desc = "A traditional Adhomian incense burner with blue and yellow suns depicted on the front. The metal cover is blackened from use, and there appear to be unclear etchings on the inside."
+	icon = 'icons/obj/custom_items/nasira_burner.dmi'
+	icon_state = "burner"
+	drop_sound = 'sound/items/drop/glass.ogg'
+	pickup_sound = 'sound/items/pickup/glass.ogg'
+	var/matchmsg = "USER lights NAME with their FLAME."
+	var/lightermsg = "USER manages to awkwardly light NAME with FLAME."
+	var/zippomsg = "With a flick of their wrist, USER lights NAME with their FLAME."
+	var/weldermsg = "USER lights NAME with FLAME. That looked rather unsafe!"
+	var/ignitermsg = "USER fiddles with FLAME, and eventually manages to light NAME."
+	var/genericmsg = "USER lights NAME with their FLAME."
+	var/lit = FALSE
+
+/obj/item/fluff/nasira_burner/Destroy()
+	STOP_PROCESSING(SSprocessing, src)
+	return ..()
+
+/obj/item/fluff/nasira_burner/examine(mob/user)
+	..(user)
+	if(lit)
+		to_chat(user, "\The [src] is currently lit.")
+
+/obj/item/fluff/nasira_burner/proc/light(var/lighting_text = "[usr] lights \the [name].")
```

Comment:
```suggestion
/obj/item/fluff/nasira_burner/proc/light(var/lighting_text)
```
Just leave this undefined, using [usr] in a place that you don't need to should be avoided